gnc/navigator_perception/include/navigator_vision_lib/model.h

```diff
@@ -0,0 +1,16 @@
+#ifndef MODEL_H
+#define MODEL_H
+#include <string>
+#include <vector>
+
+class Model
```

Comment:
I think it would be nice to make it general for different objects, but model is still a bad name. I'll rename it to PerceptionModel